content/v2.0/reference/cli/influx/repl/_index.md

```diff
@@ -19,9 +19,10 @@ from which you can run Flux commands.
 influx repl [flags]
 ```
 
-{{% note %}}
 Use **ctrl + d** to exit the REPL.
-{{% /note %}}
```

Comment:
This needs to be formatted as a note. It wasn't before and people always missed it and would submit multiple issues asking how to exit out of the REPL.

Reply:
Force pushed that change—